src/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react'
+import { questionnaires } from '../data/questionnaires'
+
+export default function Questionnaire() {
+  const currentQuestionnaire = questionnaires[0]
+  const { questions } = currentQuestionnaire
+  const [currentQuestionIndex, setcurrentQuestionIndex] = useState(0)
+  const [answers, setAnswers] = useState([])
+  const [result, setResult] = useState(0)
+
+  useEffect(() => {
+    setResult(countYes(answers))
+  }, [answers])
+
+  return (
+    <>
+      {currentQuestionIndex < questions.length ? (
+        <>
```

Comment:
Is the second fragment needed?

Reply:
The second fragment is necessary, but the first one wasn't. I erroneously thought I needed to start the return with JSX code, but I can simply start with the ternary in "normal" JS. I simplified the code now. Thanks for the input!